assets/chezmoi.io/docs/reference/templates/doppler/dopplerJson.md

```diff
@@ -0,0 +1,15 @@
+# `dopplerJson` *key* [*application-name* [*project-id* [*organization-id*]]]
+
+`dopplerJson` returns the plaintext secret from [Doppler](https://www.doppler.com) using `doppler
+secrets get --plain`, however it is a requirement that this secret be structured as `json`.
```

Comment:
I don’t use Doppler, so I can’t be 100% sure that this is right, but I would change this function as described in the [secrets access guide](https://docs.doppler.com/docs/accessing-secrets#getting-values) for getting multiple secrets: `doppler secrets get DOPPLER_PROJECT DOPPLER_CONFIG --json --no-file` or `doppler secrets download`.

Otherwise, I think we should drop `dopplerJson`, because it could be replaced with `{{ doppler key project organization | fromJson }}`.

Reply:
sounds good :smile:  
I'll implement it so that `dopplerJson` pulls all the secrets for the project and give it a few tests. Obviously this would change the template API to:  
```{{ (dopplerJson "project" "config").PASSWORD }}```

Reply:
UPDATE: I'll change it to `dopplerProjectJson` as mentioned below [here](https://github.com/twpayne/chezmoi/pull/3134#issuecomment-1659663407)